src/rgw/rgw_rados.cc

```diff
@@ -8965,6 +8986,52 @@ uint64_t RGWRados::next_bucket_id()
   return ++max_bucket_id;
 }
 
+RGWRados *RGWStoreManager::init_storage_provider(CephContext *cct, bool use_gc_thread, bool use_lc_thread,
+						 bool quota_threads, bool run_sync_thread, bool run_reshard_thread, bool use_metacache, bool use_datacache)
+{
+  RGWRados *store = NULL;
```

Comment:
RGWRados* store {nullptr};

Reply:
but wait;  why is this D3N commit changing anything to do with how we initialize RGWRados?

Reply:
> RGWRados* store {nullptr};
done.
the change is an addition of the configuration toggle to enable the data cache `.set_use_datacache(use_datacache)`

Reply:
> but wait; why is this D3N commit changing anything to do with how we initialize RGWRados?

the change is an addition of the configuration toggle to enable the data cache `.set_use_datacache(use_datacache)`